Dockerfile.travis

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Shift Devices AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:stretch
+ENV DEBIAN_FRONTEND noninteractive
+WORKDIR /app
+COPY . /app
+
+RUN apt update && apt install -y curl gnupg2 git gcc g++ make coreutils
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt install -y nodejs
+RUN apt install -y npm
+
+RUN echo "deb http://deb.debian.org/debian sid main" > /etc/apt/sources.list.d/unstable.list
+RUN apt update
+RUN apt install -y -t unstable android-sdk android-sdk-platform-23 default-jre
+RUN apt install -y -t unstable openjdk-8-jdk-headless openjdk-8-jre-headless
+RUN apt install -y -t unstable android-platform-tools-base android-sdk 
+RUN apt install -y -t unstable android-sdk-helper
+# Echo the sha1sum of the license text to these files to ensure gradle won't
+# complain about needing to accept a license text. Without this - the build breaks.
+RUN echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd" > "/usr/lib/android-sdk/licenses/android-sdk-preview-license"
+RUN echo -e "\nd56f5187479451eabf01fb78af6dfcb131a6481e" >> "/usr/lib/android-sdk/licenses/android-sdk-license"
+
+ENV JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64/"
+ENV ANDROID_HOME="/usr/lib/android-sdk" 
+ENV PATH=/usr/lib/jvm/java-8-openjdk-amd64/bin/:$PATH
+
+RUN mkdir -p /opt/src/2FA-app/
+ADD . /opt/src/2FA-app/
+RUN cd /opt/src/2FA-app/
+
+RUN npm install -g cordova
+RUN npm install -g buffer-reverse
+RUN npm install -g browserify
+
+RUN cordova prepare
+RUN npm install
+RUN npm install bitcore-lib
+
+RUN cordova platform add android
```

Comment:
Add iOS?
I believe travis has OSX instances such that it may be possible to check the iOS compilation too.